src/index.css

```diff
@@ -1,16 +1,16 @@
 body {
-  font-family: "SangBleu OG Serif Light Regular";
+  font-family: "helvetica neue";
```

Comment:
`font-family: "helvetica neue", sans-serif;`

On windows/linux "helvetica neue" isn't included, so we need a fallback. we can set fallbck to be `sans-serif`. It will be something like "arial".

Other options include:
1. buying the font (35£)
2. using a free alternative https://www.typewolf.com/top-10-helvetica-alternatives

perhaps we just use "serif" as a fallback for now ,then cross the bridge when we come to it?

Reply:
ah Typewolf <3

problem is I'd been through all his Helvetica alternatives in this article and wasn't able to find anything that was free. if you know anything Neo-Grotesquey and free then let's whack it in. for now I've added in the `sans-serif` as the fallback